Automattic/vip
# Fix issue with missing CSV output values

## Description

This fixes an issue with CSV output where values would be empty because `formatData` was changing the field names instead of using a formatter to set the header display.

## Pull request checklist


## New release checklist


## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run something like a `slowlogs` command that has fields with multiple words and make sure it is showing values for those fields
